bibfile.bib

```diff
@@ -84,3 +84,23 @@ @misc{NASADSOCTestResult
   note      = {[Online; accessed 3. May 2024]},
   url       = {https://www.jpl.nasa.gov/news/nasas-tech-demo-streams-first-video-from-deep-space-via-laser}
 }
+
+@misc{RadioWaveTravel,
+  title   = {{What{'}s the Difference between how HF and VHF/UHF radio waves travel?}},
+  journal = {Barrett},
+  year    = {2018},
+  month   = sep,
+  urldate = {2018-09-25},
+  note    = {[Online; accessed 26. Apr. 2024]},
+  url     = {https://www.barrettcommunications.com.au/news/whats-the-difference-between-how-hf-and-vhf-uhf-radio-waves-travel/}
+}
+
+@misc{SRRRange,
+  title   = {{How Far Can I Talk?}},
+  journal = {Quality Two-Way Radios},
+  year    = {2010},
+  month   = may,
+  urldate = {2010-05-22},
+  note    = {[Online; accessed 21. Feb. 2024]},
+  url     = {https://quality2wayradios.com/store/radio-range-distance}
+}
```

Comment:
You need to actually `@` (eg. `@RadioWaveTravel`) the citations in the doc, otherwise they will not be rendered in the bibliography. Put the citations after the period.